runtime/common/src/lib.rs

```diff
@@ -133,14 +133,25 @@ macro_rules! decl_common_types {
             EnsureProportionAtLeast<AccountId, TechnicalCommitteeInstance, 1, 1>,
         >;
 
-        // Advisory committee vote proportions
-        // At least 50%
-        type EnsureRootOrHalfAdvisoryCommittee = EitherOfDiverse<
+        // Advisory Committee vote proportions
+        // More than 33%
+        type EnsureRootOrMoreThanThirtyPercentAdvisoryCommittee = EitherOfDiverse<
```

Comment:
Rename it to more than 1/3 ?

Reply:
6d491cb78116b74f8ec71e8416d7bab4701a8da2 :pray: